.github/docs_pr_change_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+<!-- Describe the changes made in this PR -->
+
+## Checklist
+- [ ] This change matches the [docs style guide](https://goteleport.com/docs/contributing/documentation/style-guide/)
+- [ ] I viewed the changes on the dev server (`yarn dev`) to confirm they render without error
+- [ ] The terms used in this change correspond to those in [Teleport Core Concepts](https://goteleport.com/docs/core-concepts)
+- [ ] I spellchecked the changes
+- [ ] I referenced any issues this PR will close
```

Comment:
If we wanted to make this shorter, we could also remove this item, since it doesn't affect the docs as much as our own internal processes (i.e., it would take some extra time to close outdated issues)

Reply:
Sure.

```suggestion
```